RosBE-Unix/makepackage.sh

```diff
@@ -56,8 +56,10 @@ if ! [ -d "../../RosBE/Tools/" ]; then
 fi
 
 if ! svn info >& /dev/null; then
```

Comment:
I'd be in favour of totally dropping SVN support. There are no reasons (IMHO) to keep it.
This is even what you did on line 97. So, let's be consistent.

Reply:
Makes sense, dropped `svn` support in 64994ed